Packages/Tracking/Core/Runtime/Scripts/LeapServiceProvider.cs

```diff
@@ -964,8 +964,7 @@ protected void createController()
             }
 
             string serialNumber = _multipleDeviceMode != MultipleDeviceMode.Disabled ? SpecificSerialNumber : "";
-
-            _leapController = new Controller(serialNumber.GetHashCode(), _serverNameSpace, _multipleDeviceMode != MultipleDeviceMode.Disabled);
+            _leapController = new Controller(0, _serverNameSpace, _multipleDeviceMode != MultipleDeviceMode.Disabled);
```

Comment:
this should remain serialNumber.GetHashCode() as it has been fixed by the above line on develop :)